README.md

```diff
@@ -28,13 +28,13 @@ kraft build --target nginx-qemu-x86_64-initrd
 Once built, you can instantiate the unikernel via:
 
 ```console
-kraft run --target nginx-qemu-x86_64-initrd --initrd ./ -p 8080:80
+kraft run --target nginx-qemu-x86_64-initrd --initrd ./rootfs/ -p 8080:80
```

Comment:
```suggestion
kraft run --target nginx-qemu-x86_64-initrd --initrd ./rootfs -p 8080:80
```
This will fail for now (kraftkit on staging). Should be fixed by kraftkit, but remove it just to be safe.